src/helium_follower_service.erl

```diff
@@ -133,26 +146,38 @@ find_gateway(Chain, Ctx, Req) ->
         {ok, GwInfo} ->
             {Location, Region} =
                 case blockchain_ledger_gateway_v2:location(GwInfo) of
-                   undefined -> {<<>>, undefined};
-                   H3 ->
-                       case blockchain_region_v1:h3_to_region(H3, Ledger) of
-                            {ok, R} ->  {h3:to_string(H3), normalize_region(R)};
+                    undefined -> {<<>>, undefined};
+                    H3 ->
+                        case blockchain_region_v1:h3_to_region(H3, Ledger) of
+                            {ok, R} -> {h3:to_string(H3), normalize_region(R)};
                             _ -> {h3:to_string(H3), undefined}
                         end
-               end,
-            {ok, #follower_gateway_resp_v1_pb{
-                height=Height,
-                address = PubKeyBin,
-                location=Location,
-                owner = blockchain_ledger_gateway_v2:owner_address(GwInfo),
-                staking_mode =  blockchain_ledger_gateway_v2:mode(GwInfo),
-                gain = blockchain_ledger_gateway_v2:gain(GwInfo),
-                region = Region
-            }, Ctx}
+                end,
+            {ok,
+                #follower_gateway_resp_v1_pb{
+                    height = Height,
+                    result = {info, #gateway_info_pb{
+                        address = PubKeyBin,
+                        location = Location,
+                        owner = blockchain_ledger_gateway_v2:owner_address(GwInfo),
+                        staking_mode = blockchain_ledger_gateway_v2:mode(GwInfo),
+                        gain = blockchain_ledger_gateway_v2:gain(GwInfo),
+                        region = Region
+                    }}
+                },
+                Ctx};
+        _ ->
+            ErrorResult = {error, #follower_error_pb{type = {not_found, #gateway_not_found_pb{address = PubKeyBin}}}},
```

Comment:
looking at this again, given that the responses are not attested ( as they are with the gateway service) maybe just returning a 'not_found' response here is sufficient,  this error format with a type and sub record ( gateway_not_found_pb ) is maybe overkill ?  

On the gateway service as the responses were attested we did aim to have the response fully self contained....

I know this is late in the day etc and the proto has already be merged so feel free to ignore but

Reply:
i'm inclined to leave it as is since this the lifetime on this node solution is limited; if we decide to keep the `follower.proto` API for the off-chain metadata access we can benefit from _that_ being similarly attested or we can scrap the whole thing, retire the `follower.proto` altogether and define a new, cleaner API for the next thing that serves this data.